docs/cpp/cpp-ide.md

```diff
@@ -1,42 +1,44 @@
 ---
 Order: 9
 Area: cpp
-TOCTitle: Editing
+TOCTitle: Editing and Navigating
 ContentId: 61D63E54-67E2-4743-B5CB-C6E7F582982A
-PageTitle: Edit and navigate C++ code in Visual Studio Code
-DateApproved: 7/25/2019
+PageTitle: Edit and navigate
```

Comment:
I see - I updated the Page title to have more context around C++ and VS Code. I figured it being under the C++ tab in docs would indicate so to users, but I now called it out explicitly just in case.